lib/adapters/http.js

```diff
@@ -664,7 +664,8 @@ function HttpPouch(opts, callback) {
 
       var keysAsString =
         'keys=' + encodeURIComponent(JSON.stringify(opts.keys));
-      if (keysAsString.length + params.length + 1 <= MAX_URL_LENGTH) {
+      // FIX for sync gateway, which does not accept multiple keys on a GET request
+      if (false) { //keysAsString.length + params.length + 1 <= MAX_URL_LENGTH) {
```

Comment:
You're right I missed that : http://developer.couchbase.com/mobile/develop/references/couchbase-lite/rest-api/database/get-all-docs/index.html

I believe though the ```keys``` parameter is simply ignored.